referrer-policy/no-referrer-policy/no-referrer-policy.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Check that subresource gets the referrer URL when no explicit
```

Comment:
I think you'll need to be a little more structured with the titles, as there's a huge set of things we'll need to test. Something like "HTTP -> HTTP: Subresource" might make sense, and would allow "HTTP -> HTTPS: Subresource"/"HTTP -> HTTPS: Navigation", etc in the future.